test/features/users/author_control_page_test.rb

```diff
@@ -0,0 +1,37 @@
+require 'test_helper'
+
+feature 'Author control page' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.